lib/tasks/health.rake

```diff
@@ -117,6 +117,12 @@ namespace :health do
     )
   end
 
+  desc "Queue Retrieve Enrollments"
+  task queue_retrieve_enrollments: [:environment, "log:info_to_stdout"] do
+    user = User.setup_system_user
+    Health::UpdatePatientEnrollmentsJob.perform_later(user)
```

Comment:
This should be `perform_now` since we're calling it explicitly in a rake task/cron container.